src/Alerts/atom/Modal.tsx

```diff
@@ -16,7 +16,10 @@ const Container = styled.div`
   align-items: center;
   justify-content: center;
   background-color: hsl(0, 0%, 0%, 0.2);
```

Comment:
I wasn't sure about this when we added the initial version but this color is incorrect. It's more of a grey with a blueish hue. Zeplin says: `hsla(202, 33%, 95%, 0.51)`

Reply:
Thank you for making this correction.
I have updated.